phoenix-protocol/src/main/PTable.proto

```diff
@@ -52,11 +52,12 @@ message PColumn {
 
 message PTableStats {
   required bytes key = 1;
-  repeated bytes values = 2;
+  optional bytes guidePosts = 2;
```

Comment:
I'm leaning toward having this release be 5.0. We can add some code MetaDataRegionObserver.postOpen() that does a checkAndPut on the SYSTEM.CATALOG table for the SYSTEM.STATS row where we conditionally truncate the table (i.e. invoke the code you already wrote).